include/constants/battle_config.h

```diff
@@ -72,6 +72,7 @@
 #define B_PSYWAVE_DMG               GEN_6 // Psywave's damage formula. See Cmd_psywavedamageeffect.
 
 // Move settings
+#define B_PHYSICAL_SPECIAL_SPLIT    GEN_3 // In Gen3, the move's type determines if it will do physical or special damage.
```

Comment:
The default should be greater than `GEN_3` for consistency with the rest of the default options and to not accidentally revert the split in existing hacks unexpectedly